acct-user/iipsrv/iipsrv-0-r1.ebuild

```
@@ -0,0 +1,13 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-user
+
+DESCRIPTION="User for iipsrv"
+
+ACCT_USER_ID="536"
```

```suggestion
ACCT_USER_ID=-1
```
When you are adding it to guru, please set the user id to -1